tests/test_sequence.py

```diff
@@ -751,10 +751,10 @@ def test_mappable_register():
 def test_parametrized_index_functions(
     reg, build_params, index, expected_target
 ):
+    phi = np.pi / 4
     seq = Sequence(reg, Chadoq2)
     seq.declare_channel("ch0", "rydberg_local")
     seq.declare_channel("ch1", "raman_local")
-    phi = np.pi / 4
     index_var = seq.declare_variable("index", dtype=int)
     seq.target_index(index_var, channel="ch0")
```

Comment:
Can you add a test where you target an array instead of a single item? Plus, one where the array is subject to some operation like `var_array = var_array + 2` or something like that.

Reply:
Alright, I will do this; it will require changing the code to accept iterables of parametrized values.

Reply:
Actually, I think this should be in a separate PR. This is a new feature, and is not part of the string variables removal.

Reply:
Actually, there will be no need to change the code. I thought meant arrays like `[1, variable, 2]`, but in fact, only so-called variable arrays.